x2-signed-by-x1.yaml

```diff
@@ -25,3 +25,10 @@ certificate-profile:
   policies:
     - oid: 2.23.140.1.2.1
     - oid: 1.3.6.1.4.1.44947.1.1.1
+skip-lints:
+  # We don't use our roots to sign OCSP responses, so we don't need the digitalSignature key usage.
+  - n_ca_digital_signature_not_set
+  # This cert is a cross-sign, so not setting an EKU is fine (Mozilla Root Program §5.3).
```

Comment:
Please also cite BRs 1.7.1 7.1.2.2.g:
> For Subordinate CA Certificates *that will be used to issue TLS certificates*, the value id-kp-serverAuth [RFC5280] MUST be present.

(emphasis mine)

Also, both this lint and the one following should be described by the same cites (Moz 5.3; BR 7.1.2.2) since they amount to the same thing. n_sub_ca_eku_missing is described with reference to Technically Constained Signing Certificates (TCSCs), but probably upstream we should update it to reference 7.1.2.2.

Reply:
Done. I made some of the other comments more verbose and explicit as well.